server/src/main/java/org/candlepin/async/tasks/ExportJob.java

```diff
@@ -113,16 +116,14 @@ public static JobBuilder scheduleExport(
         final Map<String, String> extensionData
     ) {
         return JobBuilder.forJob(JOB_KEY)
```

Comment:
**A NOTE, BUT NOT FOR THIS PR:**
Not overly worried if this gets changed, but, the method scheduleExport is a little misleading as it isn't really scheduling anything; it is just creating a builder. A better name... not sure. :)

Reply:
This is definitely on my radar for the second part of this. I was never terribly thrilled with scheduling a job by returning an object from a resource and letting a filter pick it up later.

Need to come up with something that's easy to use and a little more clear in the intent.